src/Dropdown/Dropdown.js

```diff
@@ -307,37 +316,50 @@ const StyledDropdownItem = createComponent({
     role: 'button',
   }),
   as: Box,
-  style: ({ disabled, theme }) => css`
+  style: ({ disabled, theme, icon, iconProps }) => css`
     display: block;
-    width: calc(100% + 2rem);
+    flex: 1;
     opacity: ${disabled ? 0.3 : 1};
     pointer-events: ${disabled ? 'none' : 'initial'};
     user-select: ${disabled ? 'none' : 'initial'};
     text-decoration: none;
-    color: inherit;
+    color: ${theme.colors.greyDarkest};
     cursor: pointer;
-    margin: 0 calc(-1rem);
-    padding: 0.25rem 1rem;
+    margin: 0;
+    padding: 8px;
     transition: 125ms background;
     outline: none;
     appearance: none;
     border: 0;
+    border-radius: ${theme.radius}px;
     font: inherit;
+    font-size: 14px;
```

Comment:
Can we add `font-weight: 500` per the Figma specs since the text seems a little thin in the deploy preview?